google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RateLimitingServerStreamingCallable.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import com.google.api.gax.grpc.GrpcResponseMetadata;
+import com.google.api.gax.rpc.ApiCallContext;
+import com.google.api.gax.rpc.ResponseObserver;
+import com.google.api.gax.rpc.ServerStreamingCallable;
+import com.google.api.gax.rpc.StreamController;
+import com.google.bigtable.v2.MutateRowsResponse;
+import com.google.bigtable.v2.ServerStats;
+import com.google.cloud.bigtable.data.v2.stub.SafeResponseObserver;
+import com.google.common.base.Preconditions;
+import com.google.common.util.concurrent.RateLimiter;
+import io.grpc.Metadata;
+import java.time.Instant;
+import javax.annotation.Nonnull;
+
+public class RateLimitingServerStreamingCallable<RequestT, ResponseT>
```

Comment:
I'm not sure if metrics/ is the right place for this class. Maybe you can just move it under stub/.